### PR TITLE
Create bulk-scan-shared-infrastructure.json

### PR DESCRIPTION
**Pull request by jrjohnson87**
Resolves # . (This is applicable only if this pull request relates to a GitHub issue, delete the line otherwise)

Notes:
*
*
*

### CODE REVIEW

File: terraform-infra-approvals/bulk-scan-shared-infrastructure.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "resources": [
+    {"type": "azurerm_subnet_network_security_group_association"},
+    {"type": "azure_security_group"}
```
Comment (timja) [inline terraform-infra-approvals/bulk-scan-shared-infrastructure.json]: typo?

Comment (jrjohnson87) [reply terraform-infra-approvals/bulk-scan-shared-infrastructure.json]: I think it is `azure_security_group`, looking at the Jenkins error:

`[{managed azurerm_subnet_network_security_group_association subnet_nsg_association {azurerm } {app-gateway.tf 138}} {managed azure_security_group bulkscan {azure } {app-gateway.tf 108}}]`

Comment (timja) [reply terraform-infra-approvals/bulk-scan-shared-infrastructure.json]: it shouldn't be that's the legacy provider which we don't use

Comment (jrjohnson87) [reply terraform-infra-approvals/bulk-scan-shared-infrastructure.json]: Ah cool....updated the TF in the bulk-scan repo and update here. Cheers 👍

**Pull request merged**